knowledge-base/notification-single-instance-per-app.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This could be linked from somewhere in the conceptual docs too

Reply:
It is already linked in the Notification>Overview article in a tip at the end. Removed the kb.